yarn.lock

```diff
@@ -5448,7 +5500,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@popperjs/core@npm:^2.11.8, @popperjs/core@npm:^2.9.0":
```

Comment:
`floating-ui` shall be a replacement for popper, if I understand it correctly.

Ideally we should be able to get rid of it as it is used only in `@trezor/connect-explorer-theme`

Reply:
That would be more than welcomed, but it needs some refactoring here `packages/connect-explorer-theme/src/utils/use-popper.ts`.

Reply:
https://github.com/trezor/trezor-suite/issues/11695